### PR TITLE
Weblate client documentation reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: docs/wlc.rst [hunk 1]
```diff
@@ -150,56 +150,56 @@ Currently the following subcommands are available:
...
 
     .. option:: --overwrite
 
-        Overwrite existing translations on upload.
+        Overwrite existing translations upon uploading.
 
     .. option:: --input
 
-        File where to read content, if not specified file is read from stdin.
+        File from which content is read, if left unspecified the input is read from stdin.
```
Comment (comradekingu) [inline docs/wlc.rst]: Not sure this should be file 
```suggestion
        File from which content is read, if left unspecified it is read from stdin.
```

File: docs/wlc.rst [hunk 2]
```diff
@@ -150,56 +150,56 @@ Currently the following subcommands are available:
...
 
-        Convert file format, if not specified no conversion happens on server
-        and file is downloaded as is in the repository.
+        Converts file format, if unspecified no conversion happens on the server
+        and the file is downloaded as is to the repository.
 
     .. option:: --output
 
-        File where to store output, if not specified file is printed to stdout.
+        Specifies file to save output in, if left unspecified output is printed to stdout.
```
Comment (comradekingu) [inline docs/wlc.rst]:
```suggestion
        Specifies file to save output in, if left unspecified it is printed to stdout.
```

File: docs/wlc.rst [hunk 3]
```diff
@@ -36,61 +36,61 @@ Commands actually indicate which operation should be performed.
...
+using :ref:`api`. The command-line utility can be invoked as :command:`wlc` and is
 built on :mod:`wlc`.
 
 Global options
 --------------
 
-The program accepts the following global options, which must be entered before subcommand.
+The program accepts the following global options, which must be entered before any subcommand.
 
 .. option:: --format {csv,json,text,html}
```
Comment (comradekingu) [inline docs/wlc.rst]:
```suggestion
.. option:: --format {CSV,JSON,text,HTML}
```

File: docs/wlc.rst [hunk 4]
```diff
@@ -36,61 +36,61 @@ Commands actually indicate which operation should be performed.
 Description
 +++++++++++
 
-Weblate Client is Python library and command line utility to manage Weblate remotely
-using :ref:`api`. The command line utility can be invoked as :command:`wlc` and is
+Weblate Client is a Python library and command-line utility to manage Weblate remotely
+using :ref:`api`. The command-line utility can be invoked as :command:`wlc` and is
 built on :mod:`wlc`.
 
 Global options
```
Comment (comradekingu) [inline docs/wlc.rst]:
```suggestion
Instance wide options
```

File: docs/wlc.rst [hunk 5]
```diff
@@ -36,61 +36,61 @@ Commands actually indicate which operation should be performed.
...
+Weblate Client is a Python library and command-line utility to manage Weblate remotely
+using :ref:`api`. The command-line utility can be invoked as :command:`wlc` and is
 built on :mod:`wlc`.
 
-Global options
+Instance wide options
 --------------
 
-The program accepts the following global options, which must be entered before subcommand.
+The program accepts the following global options, which must be entered before any subcommand.
```
Comment (comradekingu) [inline docs/wlc.rst]:
```suggestion
The program accepts the following options for a whole instance, which must be entered before any subcommand.
```

File: docs/wlc.rst (hunk 3 above)
Comment (nijel) [reply docs/wlc.rst]: As this is what can be specified on the command line, I'd stick with lower case (and I'm not sure if uppercase will work here, probably not).

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**